resources/js/src/app/exceptions/ExceptionMap.js

```diff
@@ -18,6 +18,7 @@ export const exceptionMap = new Map(
         ["113", "errorBasketItemMaximumQuantityReachedForVariation"],
         ["114", "errorBasketItemMinimumQuantityNotReachedForVariation"],
         ["301", "notificationRemoveCouponMinimumOrderValueIsNotReached"],
+        ["302", "couponNoMatchingItemInBasket"],
```

Comment:
Hier solltest du dann auch noch den entsprechenden Eintrag in den .properties-Dateien ergänzen

Reply:
Die Keys dafür gibt es schon...